### PR TITLE
Add offline experience

### PR DESCRIPTION
**Pull request by aleksandr-mokrov**
CVS-157644

### CODE REVIEW

File: notebooks/tflite-selfie-segmentation/tflite-selfie-segmentation.ipynb [hunk 1]
```diff
@@ -87,20 +87,23 @@
   },
```
Comment (eaidova) [inline notebooks/tflite-selfie-segmentation/tflite-selfie-segmentation.ipynb:87]: <pre class="ql-syntax" spellcheck="false">Line #8.    if not Path("251036317-551a2399-303e-4a4a-a7d6-d7ce973e05c5.png").exists():
</pre><p>o we really want to preserve this long and automatically generated name? download_file allows to specify file name for saving</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2535/discussion/#comment-1848445160'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='19'/>

Comment (eaidova) [inline notebooks/tflite-selfie-segmentation/tflite-selfie-segmentation.ipynb:87]: <pre class="ql-syntax" spellcheck="false">Line #17.        video_width=1280,
</pre><p>why only width? is there is plans to provide height?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2535/discussion/#comment-1848445162'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='27'/>

Comment (eaidova) [inline notebooks/tflite-selfie-segmentation/tflite-selfie-segmentation.ipynb:87]: <pre class="ql-syntax" spellcheck="false">Line #9.        if not Path("CEO%20Pat%20Gelsinger%20on%20Leading%20Intel.mp4").exists():
</pre><p>the same comment about name</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2535/discussion/#comment-1848445164'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='29'/>

Comment (aleksandr-mokrov) [reply notebooks/tflite-selfie-segmentation/tflite-selfie-segmentation.ipynb:87]: renamed

Comment (aleksandr-mokrov) [reply notebooks/tflite-selfie-segmentation/tflite-selfie-segmentation.ipynb:87]: renamed

**Review by eaidova — APPROVED** [review]

**Pull request merged**